local.bib

```diff
@@ -0,0 +1,51 @@
+% bibentry template for lsst.bib in https://github.com/lsst/lsst-texmf/blob/master/texmf/bibtex/bib/lsst.bib
+
+@DocuShare{POMS,
+   author = { Mengel and White and  Podstavkov and Wiersma and Mazzacane and Herner},
+    title = {Production Operations Management System (POMS) for
+Fermilab Experiments},
+     year = 2020,
+    month = Apr,
+   handle = {POMS},
+      url = {https://www.epj-conferences.org/articles/epjconf/pdf/2020/21/epjconf_chep2020_03024.pdf } }
+
+
+@DocuShare{PanDA,
+   author = {PanDA},
+    title = {Production and Distributed Analysis},
+     year = 2021,
+    month = Jan,
+   handle = {PanDA},
+      url = {https://panda-wms.readthedocs.io/en/latest } }
+
+@DocuShare{Airflow,
+   author = { Apache },
+    title = {Apache Airflow workflow generation and monitoring software},
+     year = 2021,
+    month = Jan,
+   handle = {airflow},
+      url = {https://airflow.apache.org } }
+
+@DocuShare{DMTN-173,
+   author = {Frossie Economou},
+    title = {The Observatory Logging Ecosystem},
+     year = 2020,
+    month = Dec,
+   handle = {DMTN-173},
+      url = {https://dmtn-173.lsst.io} }
+
+@DocuShare{DMTN-181,
+   author = {Kian-Tat Lim and Yusra AlSayyad and Jim Bosch and Hsin-Fang Chiang and Richard DuBois and Robert Gruendl and Leanne Guy and Tim Jenness and Colin Slater},
+    title = {Campaign Management},
+     year = 2021,
+    month = Feb,
+   handle = {DMTN-181},
+      url = {https://dmtn-181.lsst.io/v/u-ktl-initial-draft/index.html } }
+
+@DocuShare{DMTR-311,
+   author = {Jeff Carlin},
+    title = {Characteriation Metric Report: Science Pipelines Version 22.0.0},
+     year = 2021,
+    month = Jul,
+   handle = {Metrics},
+      url = {https://dmtr-311.lsst.io } }
```

Comment:
There are multiple \cite that aren't in the local.bib (DMTR-311 may be just mismatch in handle?)

Reply:
Maybe I need to commit local.bib again.